google/cloud/security/enforcer/gce_firewall_enforcer.py

```diff
@@ -144,6 +173,14 @@ def __init__(self, gce_service, dry_run=False):
         retry_on_exception=http_retry,
         wait_exponential_multiplier=1000,
         stop_max_attempt_number=4)
+
```

Comment:
Nit: Move pylint comment and blank space above the @retry wrapper for readability.

Reply:
Done